en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/inter-component-communication.md

```diff
@@ -0,0 +1,50 @@
+# Communnication between the Components
+
+Communication between internal components of Choreo Connect (adapter, enforcer, router and apim control plane) are secured via mutual ssl.
```

Comment:
Shall we introduce this concept more so that new users can understand? We need to explain what sort of communication this is and why it is needed? We should also explain why securing this communication is important for the user to care about this.

Reply:
```suggestion
Communication between internal components of Choreo Connect (Adapter, Enforcer, Router and API-M control plane) are secured via mutual ssl.
```